app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -88,15 +88,15 @@ snippet: true
       subscription_manager_atomic_url = subscription_manager_configuration_url(@host, false)
       subscription_manager_org = @host.rhsm_organization_label
       activation_key = host_param('kt_activation_keys')
-      redhat_install_agent = host_param_true?('redhat_install_agent', true)
+      redhat_install_agent = host_param_true?('redhat_install_agent', katello_agent_enabled?)
```

Comment:
If we are exactly doing same call `host_param_true?('redhat_install_agent', katello_agent_enabled?)` for both `if and else` block then is it good to take `redhat_install_agent = host_param_true?('redhat_install_agent', katello_agent_enabled?)` out of the `if..else` ?,